src/include/migraphx/streamutils.hpp

```diff
@@ -59,6 +60,11 @@ inline stream_range_container<Range> stream_range(const Range& r)
 
 namespace detail {
 
+inline void stream_write_value_impl(rank<3>, std::ostream& os, const migraphx::value::binary&)
+{
+    os << "{binary_object}";
```

Comment:
This is nice doing it generically, I would just write a custom `operator<<` for the operator.

On `code_object_op` I usually show the size of the binary. Maybe we could do `os << "{binary_object:" << binary.size() << "}"`.

Reply:
Added `size`  in the prints.

Reply:
> > I would just write a custom operator<< for the operator.
> 
> I can do it, but unlike code_object_op, convolution has containers i.e. padding, dilations etc. in its fields.

No, dont do that. I was just saying what I have done in the past. I guess the "would" is a little ambiguous there. Doing this generically is much better.